hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/7_0_0/5610-allow-broker-other-than-jacksonmessageconverter .yaml

```
@@ -0,0 +1,5 @@
+---
+type: add
+issue: 5610
+title: "Before message broker always used `JacksonMessageConverter`. A configuration option was added to allow not to use
+    it, so other converters can be selected depending on the message type."
```

It looks like change-log file should be moved to `7_2_0` folder and renamed - there is trailing space at the name of the file (`5610-allow-broker-other-than-jacksonmessageconverter .yaml -> 5610-allow-broker-other-than-jacksonmessageconverter.yaml`)
And small suggestion regarding changelog title:
```suggestion
title: "Before, message broker always used `JacksonMessageConverter`. Added a configuration option that allows
    to disable it so other converters can be configured depending on the message type."
```

---

Fixed. Thanks!